src/mlpack/methods/reinforcement_learning/environment/pendulum.hpp

```diff
@@ -53,7 +54,7 @@ class Pendulum
     { /* Nothing to do here. */ }
 
     //! Modify the internal representation of the state.
-    arma::colvec& Data() { return data; }
+    arma::colvec& Data() { return convertedData; }
```

Comment:
Do you think we should add a new setter instead of changing the old one? Actually I'm not really sure why you've made this change

Reply:
The reason is that the `Pendulum-v0` gym env gives out `3` state values:
- sin(theta)
- cos(theta)
- angular velocity

But our implementation initially only gave out
- theta
- angular velocity

So I made the changes to be consistent with gym's implementation, and `cos` and `sin` values would be more interpretable to the agent, acc to me.

> Do you think we should add a new setter instead of changing the old one?

Please have a look at the most recent change. Now I'm storing theta as an internal state, and modifying `data` to include the cos and sin values.

Reply:
Oh, now I got it. You'll have to change documentation in a couple places (like above the state and such), could you do that?

Reply:
I think there's just one such place right?

Reply:
Right, I think so too